reconcile/utils/jjb_client.py

```diff
@@ -383,6 +383,10 @@ def get_repo_url(job):
         repo_url_raw = job['properties'][0]['github']['url']
         return repo_url_raw.strip('/').replace('.git', '')
 
+    @staticmethod
+    def get_ref(job: dict) -> str:
```

Comment:
i'm going to adopt this style where i will use dict without further key/value type hints when we talk about a generic dict.